### ISSUE TITLE
Apache Parquet format

### ISSUE THREAD

**Issue by MattBlissett** [gbif/occurrence-cube#17]
> Column-oriented data format, optimized for data storage and retrieval. Increasingly used in tools like Google Big Query. Can be read by e.g. R package “arrow” (Richerson et al. 2023).

https://docs.gbif-uat.org/b-cubed/1.0/en/#format

**Comment by MattBlissett** [comment]
I think we should check the data size of typical cubes before implementing this. Several of the advantages of Parquet format (efficient for very large datasets) might not be relevant.

**Comment by peterdesmet** [comment]
I agree, the cube is designed to downsize data. If or when parquet formats are implemented for the occurrence download API, we might consider it for the SQL download API.
